fem/qfunction.hpp

```diff
@@ -47,6 +47,15 @@ class QuadratureFunction : public Vector
    QuadratureFunction(QuadratureSpaceBase *qspace_, int vdim_ = 1)
       : QuadratureFunction(*qspace_, vdim_) { }
 
+   /** @brief Create a QuadratureFunction based on the given QuadratureSpaceBase,
+       using the external data, @a qf_data. */
+   /** The QuadratureFunction does not assume ownership of the
```

Comment:
Thanks @rcarson3.
Thoughts @v-dobrev ?

Reply:
Yes, the `Vector` constructor used here assumes that `qf_data` is a host pointer -- I think it is good to clarify that here.

@kennyweiss, do you need to use this with a device pointer?

Reply:
I think it should be ok for now since we're allocating it in Sidre within Axom's `MFEMSidreDataCollection`.
If we want to allocate and use it directly on the device, we might need to add another constructor.

Reply:
@kennyweiss do you want to modify/clarify the Doxygen comment? I can also do it if you prefer. Thanks

Reply:
Thanks @pazner -- would you mind adding it?

Reply:
Done